backend/src/main/java/com/salat/briene/config/AuthTokenFilter.java

```diff
@@ -50,7 +50,7 @@ protected void doFilterInternal(@NotNull HttpServletRequest request,
     private String parseJwt(HttpServletRequest request) {
         String headerAuth = request.getHeader("Authorization");
 
-        if (StringUtils.hasText(headerAuth) && headerAuth.startsWith("Bearer ")) { // todo why Bearer?
+        if (StringUtils.hasText(headerAuth) && headerAuth.startsWith("Bearer ")) {
```

Comment:
Don't use stiring literals without varible declaration. It can be constants almost all time. Next line including "magic number".